src/nvme/linux.c

```diff
@@ -176,16 +176,18 @@ int nvme_get_log_page_padded(int fd, __u32 nsid, __u8 log_id, bool rae,
 }
 
 static int nvme_get_telemetry_log(int fd, bool create, bool ctrl, bool rae,
-				  struct nvme_telemetry_log **buf)
+				  struct nvme_telemetry_log **buf, int da, size_t *size)
```

Comment:
I'd prefer if we introduced an enum for the `da` argument, similar to `enum nvme_hmac_alg` for example.

Reply:
Fixed with latest commit.

Reply:
Fixed in commit cbfe4a46431b3791d6aaeec4f5e4a46323379d40.